docs/building.rst

```diff
@@ -647,3 +654,18 @@ in a command shell::
 
 This will also install the headers and library files in the visual studio directories to
 be picked up by libtorrent.
+
+list of macros
+--------------------
+
+The following is a list of defines that libtorrent is built with: ``BOOST_ALL_NO_LIB``, ``BOOST_ASIO_ENABLE_CANCELIO``, ``BOOST_ASIO_HAS_STD_CHRONO``, ``BOOST_MULTI_INDEX_DISABLE_SERIALIZATION``, ``BOOST_NO_DEPRECATED``, ``BOOST_SYSTEM_NO_DEPRECATED``, ``TORRENT_BUILDING_LIBRARY``, ``TORRENT_USE_I2P=1``
+
+Make sure you define the same at complile time for your code to avoid any runtime errors and other issues.
+
+These might change in the future, so it's always best to verify these every time you upgrade to
+a new version of libtorrent. The simplest way to see the full list of macros defined is to build
+libtorrent with "--debug-configuration -d2+2" switches added to b2 command line:
+
+``b2 --debug-configuration -d2+2 toolset=msvc-14.2 link=static runtime-link=static boost-link=static variant=release``
```

Comment:
I would prefer command lines to be in "literal blocks", i.e. put a "::", blank line and indentation (there are other examples in this file)

Reply:
Yeah, I see. Indentation is what I missed in front of the command line, because I tried to use double colon and it wasn't working.